docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_webhook.go

```diff
@@ -65,26 +70,42 @@ A webhook will automatically be served that calls this defaulting.
 The `Default` method is expected to mutate the receiver, setting the defaults.
 */
 
-var _ webhook.Defaulter = &CronJob{}
+type CronJobCustomDefaulter struct{}
 
-// Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *CronJob) Default() {
-	cronjoblog.Info("default", "name", r.Name)
+var _ webhook.CustomDefaulter = &CronJobCustomDefaulter{}
 
-	if r.Spec.ConcurrencyPolicy == "" {
-		r.Spec.ConcurrencyPolicy = AllowConcurrent
+// Default implements webhook.CustomDefaulter so a webhook will be registered for the type
```

Comment:
nit:

```suggestion
// Default implements webhook.CustomDefaulter so a webhook will be registered for the type CronJob 
```

Because the type was inferred when dealing with registering the webhook when we did this on the type, the type is now only known when registering the webhook with the manager.  

Maybe adding this on the comment will explain that relationship.